queries/css/highlights.scm

```diff
@@ -1,72 +1,88 @@
-"@media" @keyword
-"@import" @include
-"@charset" @keyword
-"@namespace" @keyword
-"@supports" @keyword
-"@keyframes" @keyword
-(at_keyword) @keyword
-(to) @keyword
-(from) @keyword
-(important) @keyword
+[
+ "@media"
+ "@import"
+ "@charset"
+ "@namespace"
+ "@supports"
+ "@keyframes"
+ (at_keyword)
+ (to)
+ (from)
+ (important)
+ ] @keyword
 
 (comment) @comment
 
-(tag_name) @type
-(nesting_selector) @type
-(universal_selector) @type
+[
+ (tag_name)
+ (nesting_selector)
+ (universal_selector)
+ ] @type
 
 (function_name) @function
 
-"~" @operator
-">" @operator
-"+" @operator
-"-" @operator
-"*" @operator
-"/" @operator
-"=" @operator
-"^=" @operator
-"|=" @operator
-"~=" @operator
-"$=" @operator
-"*=" @operator
+[
+ "~"
+ ">"
+ "+"
+ "-"
+ "*"
+ "/"
+ "="
+ "^="
+ "|="
+ "~="
+ "$="
+ "*=" 
+ "and"
+ "or"
+ "not"
+ "only"
+ ] @operator
 
-"and" @operator
-"or" @operator
-"not" @operator
-"only" @operator
 
 (attribute_selector (plain_value) @string)
 (pseudo_element_selector (tag_name) @property)
 (pseudo_class_selector (class_name) @property)
 
-(class_name) @property
-(id_name) @property
-(namespace_name) @property
-(property_name) @property
-(feature_name) @property
-(attribute_name) @property
+[
+ (class_name)
+ (id_name)
+ (namespace_name)
+ (property_name)
+ (feature_name)
+ (attribute_name)
+ ] @property
 
 
 ((property_name) @type
- (#match? @type "^--"))
+                 (#match? @type "^--"))
 ((plain_value) @type
- (#match? @type "^--"))
+               (#match? @type "^--"))
```

Comment:
are those tabs required ? i dont mind but it looks weird on github

Reply:
Good catch, it was just vim trying to be smart with the indentation :smile: